strictdoc/backend/excel/import_/excel_sheet_proxy.py

```diff
@@ -0,0 +1,70 @@
+from typing import List
+
+import openpyxl
+import xlrd
+
+from strictdoc.helpers.auto_described import auto_described
+
+
+@auto_described
+class ExcelSheetProxy:
+    """This proxy class allows to open either xls(xlrd) or xlsx(openpyxl) with the same interface"""
+
+    def __init__(self, file: str):
+        self.file = file
+        if file.endswith(".xlsx"):
+            self.workbook = openpyxl.load_workbook(file)
+            self.sheet = self.workbook.active
```

Comment:
Should handle the case that `.active` returns `None`. Raise a `RuntimeError` or something.